PythonRegression/tests/features/steps/api_test_steps.py

```diff
@@ -73,16 +73,16 @@ def call_getTrytes(step):
 def check_trytes(step):
     response = responses['getTrytes'][config['nodeId']]
     if 'trytes' in response:
-        assert response['trytes'][0] == testTrytes, "Trytes do not match"
+        assert response['trytes'][0] == testTrytes, "Trytes do not match: \n{}".format(response['trytes'][0])
 
 
 ###
 #Test Add and Remove Neighbors
   
-@step(r'2 neighbors are added with "([^"]*)" on "([^"]*)"')
-def add_neighbors(step,apiCall,nodeName):
+@step(r'2 neighbors are added with "([^"]*)" on "([^"]*)" in "([^"]*)"')
+def add_neighbors(step,apiCall,nodeName,machine):
```

Comment:
didn't we say we are going to get rid of explicitely calling machines like this.
Open an issue

Reply:
https://github.com/iotaledger/iri-regression/issues/28